ml-proto/host/format.mli

```diff
@@ -0,0 +1,7 @@
+open Sexpr
+
+val func_type : Types.func_type -> sexpr
+
+val expr : Kernel.expr -> sexpr
+val module_ : Kernel.module_ -> sexpr
```

Comment:
I'd been thinking it'd make sense just to put it in `Sexpr`, renaming the functions `of_module`/`of_expr`.